docs/dev/dev_environment.md

```diff
@@ -73,7 +76,7 @@ nautobot-server migrate
 ```
 
 !!! note
-    If you want to develop on the latest develop branch of Nautobot, run the following command: `poetry add --optional git+https://github.com/nautobot/nautobot@develop`. After the `@` symbol must match either a branch or a tag.
```

Comment:
Can you revert these formatting changes? The MKDocs admonitions we use for these requires the indentation.

Reply:
@smk4664 we really need to get the MKDocs admonitions work with standard markdown formatters.

Reply:
This is just from a save that had auto format on, which is still my preferred method.